--- a/libntfs-3g/Makefile.in	2022-10-20 18:35:11.000000000 +0300
+++ b/libntfs-3g/Makefile.in	2026-04-01 13:33:14.220911338 +0300
@@ -1170,13 +1170,13 @@
 # We may need to move .so files to root
 # And create ldscript or symbolic link from /usr
 install-exec-hook: install-rootlibLTLIBRARIES
-@INSTALL_LIBRARY_TRUE@	if [ ! "$(rootlibdir)" -ef "$(libdir)" ]; then \
+@INSTALL_LIBRARY_TRUE@	if [ "$(rootlibdir)" != "$(libdir)" ]; then \
 @INSTALL_LIBRARY_TRUE@		$(MV) -f "$(DESTDIR)/$(libdir)"/libntfs-3g.so* "$(DESTDIR)/$(rootlibdir)";  \
 @INSTALL_LIBRARY_TRUE@	fi
-@GENERATE_LDSCRIPT_TRUE@@INSTALL_LIBRARY_TRUE@	if [ ! "$(rootlibdir)" -ef "$(libdir)" ]; then \
+@GENERATE_LDSCRIPT_TRUE@@INSTALL_LIBRARY_TRUE@	if [ "$(rootlibdir)" != "$(libdir)" ]; then \
 @GENERATE_LDSCRIPT_TRUE@@INSTALL_LIBRARY_TRUE@		$(install_sh_PROGRAM) "libntfs-3g.script.so" "$(DESTDIR)/$(libdir)/libntfs-3g.so"; \
 @GENERATE_LDSCRIPT_TRUE@@INSTALL_LIBRARY_TRUE@	fi
-@GENERATE_LDSCRIPT_FALSE@@INSTALL_LIBRARY_TRUE@	if [ ! "$(rootlibdir)" -ef "$(libdir)" ]; then \
+@GENERATE_LDSCRIPT_FALSE@@INSTALL_LIBRARY_TRUE@	if [ "$(rootlibdir)" != "$(libdir)" ]; then \
 @GENERATE_LDSCRIPT_FALSE@@INSTALL_LIBRARY_TRUE@		$(LN_S) "$(rootlibdir)/libntfs-3g.so" "$(DESTDIR)/$(libdir)/libntfs-3g.so"; \
 @GENERATE_LDSCRIPT_FALSE@@INSTALL_LIBRARY_TRUE@	fi
 
